File: examples/python/delete/deleteOne.py
# Start Connection
from pymongo import MongoClient
from pprint import pprint
import json

connection_string = "mongodb://testuser:<PASSWORD>@localhost:27017/test?authSource=admin"


client = MongoClient(connection_string)
# End Connection

# Start select db
db = client.test
# End select db

# Start updateOne (Example 58)
result = db.inventory.delete_one({"status": "D"})
# End updateOne (Example 58)

pprint(result.raw_result)

client.close()

File: source/includes/examples/node/delete/deleteOne.js
const MongoClient = require('mongodb').MongoClient;
const assert = require('assert');

// Connection URL
const url = 'mongodb://localhost:27017/test';

// Database Name
const dbName = 'test';

// use connect method to connect to the server
MongoClient.connect(url, function(err, client){
  assert.equal(null, err);
  console.log("Connected successfully to server!");
  const db = client.db(dbName)

  // Delete a single document (example 58)
  var promise = 
  db.collection('inventory').deleteOne({ 
    status: "D" 
  })
  .then(function(result) {
    // process result
    console.log("deleteOne example result: " + result)
  })            
  // End deleteOne example
  client.close();
});

